install/v1alpha1-0.7.0/cstor-volume/volume.yaml

```diff
@@ -0,0 +1,755 @@
+      labelSelector: openebs.io/storagepoolclaim={{ .Config.StoragePoolClaim.value }}
+  post: |
+    {{/*
+    Check if enough online pools are present to create replicas.
+    If pools are not present error out.
+    Save the cstorpool's uid:name into .ListItems.cvolPoolList otherwise
+    */}}
+    {{- $replicaCount := int64 .Config.ReplicaCount.value | saveAs "rc" .ListItems -}}
+    {{- $poolsList := jsonpath .JsonResult `{range .items[?(@.status.phase=="Online")]}pkey=pools,{@.metadata.uid}={@.metadata.name};{end}` | trim | default "" | splitList ";" -}}
+    {{- $poolsList | saveAs "pl" .ListItems -}}
+    {{- len $poolsList | gt $replicaCount | verifyErr "not enough pools available to create replicas" | saveAs "cvolcreatelistpool.verifyErr" .TaskResult | noop -}}
+    {{- $poolsList | keyMap "cvolPoolList" .ListItems | noop -}}
+---
+# runTask to create cStor controller service
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cstor-volume-create-puttargetservice-default-0.7.0
+  namespace: openebs
+data:
+  meta: |
+    apiVersion: v1
+    kind: Service
+    action: put
+    id: cvolcreateputsvc
+    runNamespace: {{ .Volume.runNamespace }}
+  post: |
+    {{- jsonpath .JsonResult `{.metadata.name}` | trim | saveAs "cvolcreateputsvc.objectName" .TaskResult | noop -}}
+    {{- jsonpath .JsonResult `{.spec.clusterIP}` | trim | saveAs "cvolcreateputsvc.clusterIP" .TaskResult | noop -}}
+  task: |
+    apiVersion: v1
+    kind: Service
+    metadata:
+      labels:
+        openebs.io/controller-service: cstor-controller-svc
+        openebs.io/storage-engine-type: cstor
+        openebs.io/pv: {{ .Volume.owner }}
+      name: {{ .Volume.owner }}
+    spec:
+      ports:
+      - name: cstor-iscsi
+        port: 3260
+        protocol: TCP
+        targetPort: 3260
+      - name: mgmt
+        port: 6060
+        targetPort: 6060
+        protocol: TCP
+      selector:
+        openebs.io/controller: cstor-controller
+        openebs.io/pv: {{ .Volume.owner }}
+        app: cstor-volume-manager
+---
+# runTask to create cStorVolume
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cstor-volume-create-putcstorvolumecr-default-0.7.0
+  namespace: openebs
+data:
+  meta: |
+    apiVersion: openebs.io/v1alpha1
+    kind: CStorVolume
+    id: cvolcreateputvolume
+    runNamespace: openebs
+    action: put
+  post: |
+    {{- jsonpath .JsonResult `{.metadata.uid}` | trim | saveAs "cvolcreateputvolume.cstorid" .TaskResult | noop -}}
+    {{- jsonpath .JsonResult `{.metadata.name}` | trim | saveAs "cvolcreateputvolume.objectName" .TaskResult | noop -}}
+  task: |
+    {{- $replicaCount := .Config.ReplicaCount.value | int64 -}}
+    apiVersion: openebs.io/v1alpha1
+    kind: CStorVolume
+    metadata:
+      name: {{ .Volume.owner }}
+      labels:
+        openebs.io/pv: {{ .Volume.owner }}
+    spec:
+      targetIP: {{ .TaskResult.cvolcreateputsvc.clusterIP }}
+      capacity: {{ .Volume.capacity }}
+      iqn: iqn.2016-09.com.openebs.cstor
```

Comment:
Can we revert this back to complete `iqn` and expect @Moteesh 's code to split and use it for the sake of consistency.